CHANGELOG.latest.md

```diff
@@ -1,6 +1,2 @@
-### Dependency Updates
-* [AUTOMATIC BUMP] Updates purchases-hybrid-common to 4.18.0 (#288) via RevenueCat Git Bot (@RCGitBot)
-  * [iOS 4.21.0](https://github.com/RevenueCat/purchases-ios/releases/tag/4.21.0)
-  * [iOS 4.19.1](https://github.com/RevenueCat/purchases-ios/releases/tag/4.19.1)
-  * [iOS 4.20.0](https://github.com/RevenueCat/purchases-ios/releases/tag/4.20.0)
-* Bump fastlane from 2.212.2 to 2.213.0 (#285) via dependabot[bot] (@dependabot[bot])
+### Other Changes

```

Comment:
Should this be under `### New Features`?

Reply:
Swapped in the real changelog :D